client/index.html

```diff
@@ -95,23 +118,13 @@
                   </li>
                   <li class="divider"></li>
                   <li class="link">
-                    <a href="#">
-                      Profile
-                    </a>
-                  </li>
-                  <li class="link">
-                    <a href="#">
-                      Menu Item
-                    </a>
-                  </li>
-                  <li class="link">
-                    <a href="#">
-                      Menu Item
+                    <a href="http://www.github.com">
+                      My Github
                     </a>
                   </li>
                   <li class="divider"></li>
                   <li class="link">
-                    <a href="#">
+                    <a href="#/logout">
```

Comment:
This link should not include the #. It needs to send the user to the /logout directly on the server.